hassio-hostapd/run.sh

```diff
@@ -104,6 +105,15 @@ echo "channel=${CHANNEL}" >> ${HCONFIG}
 echo "interface=${INTERFACE}" >> ${HCONFIG}
 echo "" >> ${HCONFIG}
 
+if test ${HIDE_SSID} = true;
+then
+    echo "Hidding SSID"
+    sed -i 's/ignore_broadcast_ssid=0/ignore_broadcast_ssid=1/' ${HCONFIG}
+else
+    echo "Showing SSID"
+    sed -i 's/ignore_broadcast_ssid=1/ignore_broadcast_ssid=0/' ${HCONFIG}
```

Comment:
Same here. You can echo either `ignore_broadcast_ssid=1` or `ignore_broadcast_ssid=0`